lib/msf/core/db_manager/workspace.rb

```diff
@@ -97,11 +97,16 @@ def update_workspace(opts)
     Msf::Util::DBManager.delete_opts_workspace(opts)
 
     ::ActiveRecord::Base.connection_pool.with_connection {
-      ws_to_update = workspaces({ id: opts.delete(:id) }).first
+      ws_id = opts.delete(:id)
+      ws_to_update = workspaces({ id: ws_id }).first
       default_renamed = true if ws_to_update.name == DEFAULT_WORKSPACE_NAME
-      updated_ws = ws_to_update.update!(opts)
```

Comment:
This line prompted this change as `updated_ws` actually returns a boolean not the workspace object.